entrypoint.sh

```diff
@@ -8,26 +8,34 @@ APP_GROUP=steam
 APP_HOME=/home/$APP_USER
 
 source /includes/colors.sh
+source /includes/permissions.sh
 
 if [[ "${PUID}" -eq 0 ]] || [[ "${PGID}" -eq 0 ]]; then
-    ee ">>> Running as root is not supported, please fix your PUID and PGID!"
+    ee ">>> Running Palworld as root is not supported, please fix your PUID and PGID!"
```

Comment:
">>> Running this container as root is not supported, please fix your PUID and PGID!"

Reply:
I was trying to make a distinction between the `PUID`/`PGID` and the [`-User`](https://docs.docker.com/engine/reference/run/#user) flag when running Docker. This may just be my newness to Docker but I understood that flag to mean the _container_ runs as the specified user (or root if none specified) and gosu runs _Palworld_ as the `PUID`/`PGID`.  From there I reasoned that `PUID`/`PGID` check is to prevent _Palworld_ from running as root but the container _must_ run as root (or at least it did before this PR). 

I'll need to look more as to why but for me when I was running an older version of the container it automatically ran as steam and when I updated the container, it retained that. I know that should have been avoided by using docker-compose but the existence of #203 and comments in the discord show I am not the only one.
![image](https://github.com/jammsen/docker-palworld-dedicated-server/assets/45444205/5d2858e5-c80f-4ed9-bb61-2a921cffff9e)

To summarize, its my (mis?)understanding that the _container_ runs as root and _Palworld_ runs as `PUID`/`PGID`. I'm completely open to being corrected here, just trying to explain my line of thinking.

Reply:
Well that you have to switch from user steam to user root is down to using Portainer, in a Docker-Compose use-case you would not need to bother with that, but Portainer uses certain things and bends it to its will and persist it into some weird configuration values you need to keep in mind, no matter if its usefull or useless for you. TL;DR, thats just down to the habbits of the tool you are using, need to know this and fix it yourself.

Reply:
Regarding the -user flag, thats the User-Remapping feature of docker, while you can have any numbers of users in docker internally and having Docker installed as Root-ful and not Rootless, you can do what ever you want internally, BUT when you need to write files back to the file system, now the files are owned by root, because any container run by root-ful Docker runs as root, thats just how it is and as designed.

Enter User-Remapping, you can say i want to write files like "npm install" inside a container but thoose files would now be owned be root or UID 999, but sometimes you dont want that. Thats what the User-Remapping feature is for, the process in the host runs now under this user and group and the files coming back are owned by this user and not root. Thats the basic idea here, see the Medium article i linked you for more detailed description.